src/components/Dashboard/index.jsx

```diff
@@ -11,15 +14,21 @@ class Dashboard extends React.Component {
   You will create a function renderUpselling that returns
```

Comment:
Nice explanation and straight forward steps on what the function you pass should do. Please edit the comments before this last statement as they are not clear. I think that what you meant is that the reusable card can be used to render different information as such three properties should be passed to it that is class, title and a function that returns jsx.